librz/core/cagraph.c

```diff
@@ -131,17 +131,21 @@ static void agraph_print_edge(RzANode *from, RzANode *to, void *user) {
 }
 
 static void agraph_print_node(RzANode *n, void *user) {
-	char *encbody, *cmd;
+	char *cmd;
 	int len = strlen(n->body);
 
 	if (len > 0 && n->body[len - 1] == '\n') {
 		len--;
 	}
-	encbody = rz_base64_encode_dyn((const ut8 *)n->body, len);
-	cmd = rz_str_newf("agn \"%s\" base64:%s\n", n->title, encbody);
+	if (RZ_STR_ISEMPTY(n->body)) {
+		cmd = rz_str_newf("agn \"%s\"\n", n->title);
+	} else {
+		char *encbody = rz_base64_encode_dyn((const ut8 *)n->body, len);
+		cmd = rz_str_newf("agn \"%s\" base64:%s\n", n->title, encbody);
```

Comment:
this looks like a command injection bug. maybe is worty to open an issue?

Reply:
yes